db/migrate/20230627142246_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :users, null: false, foreign_key: true
```

Comment:
- [ ] Kindly change the references to `author` instead of `user` in the comment table as per the [diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).

Reply:
@gemmen29 
As I can see you have done the following using the same method.
![Screenshot 2023-06-27 195956](https://github.com/IsmailChangezi/blog-app-new/assets/96125730/8f69768c-283b-4859-9890-6a4e21b47bda)